operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -28,11 +28,11 @@ Mattermost numbers stable releases in the following format: **\[Version Number\]
 
 ## Objectives
 
-The goal is to deliver value to users quickly by a\) shipping fast to get features to customers quickly for experimentation/feedback, and b\) iteratively behind feature flags as a protection if any issues arise. This document outlines the principles and guidelines for how we release a new update of each product line weekly to Cloud and monthly to self-managed.
+The goal is to deliver value to users quickly by a\) shipping fast to get features to customers quickly for experimentation/feedback, and b\) iteratively behind feature flags as a protection if any issues arise. This document outlines the principles and guidelines for how we release a new update of each product line biweekly to Cloud and monthly to self-managed.
 
 ## Multi-product Release Principles
 
-The goal is not to have the same release cadence for all products, but there may be similar schedules. E.g. Playbooks and Boards plugins are pre-packed once a month with the self-managed releases, however, admins will have a config option to turn off products. As the product suite gets bigger, we need a dedicated Release Manager for each product. We can duplicate/reuse a similar build process for new plugins in the suite.
+The goal is not to have the same release cadence for all products, but there may be similar schedules. E.g. Playbooks and Boards plugins are pre-packed once a month with the self-managed releases, however, admins will have a config option to turn off products. We can duplicate/reuse a similar build process for new plugins in the suite.
```

Comment:
Same here in pre-packaging Playbooks and Boards if still accurate given they are to be shipped as built-in products.

Reply:
Monorepo didn't make it to v7.10 release, so I'll let @sbishel to confirm how this works now.

Reply:
Other than security patches, neither boards nor playbooks will cut plugins any more.